### ISSUE TITLE
Installation error on  macOS 14 && Python 3.12

### ISSUE THREAD

**Issue by Kyparos** [ManimCommunity/manim#3422]
#### Preliminaries

- [x] I have followed the latest version of the
      [installation instructions](https://docs.manim.community/en/stable/installation.html).
- [x] I have checked the [installation FAQ](https://docs.manim.community/en/stable/faq/installation.html) and my problem is either not mentioned there,
      or the solution given there does not help.

## Description of error
Error occurs while installing manim trough pip install (either `pip install manim` or `pip3 install manim`) on  macOS 14 with python 3.12.
The reason why I think it is connected to exactly this combination of OS and Python version is this [Reddit tree](https://www.reddit.com/r/manim/comments/1744e7a/comment/k4yopvz/?utm_source=share&utm_medium=web2x&context=3) and my personal experience. 

## Installation logs

<details><summary>Terminal output</summary>

[log](https://pastebin.com/LDz1FbJ5)
</details>

## System specifications

<details><summary>System Details</summary>

- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)): macOS 14 (Sonoma)
- RAM: 8GB
- Python version (`python/py/python3 --version`): 3.12
- Installed modules (provide output from `pip list`): 
```
Package                   Version
------------------------- ------------
anyio                     4.0.0
appnope                   0.1.3
argon2-cffi               23.1.0
argon2-cffi-bindings      21.2.0
arrow                     1.3.0
asttokens                 2.4.0
async-lru                 2.0.4
attrs                     23.1.0
Babel                     2.13.1
backcall                  0.2.0
beautifulsoup4            4.12.2
bleach                    6.1.0
certifi                   2023.7.22
cffi                      1.16.0
charset-normalizer        3.3.1
comm                      0.1.4
debugpy                   1.8.0
decorator                 5.1.1
defusedxml                0.7.1
executing                 2.0.0
fastjsonschema            2.18.1
fqdn                      1.5.1
idna                      3.4
ipykernel                 6.25.2
ipython                   8.16.1
isoduration               20.11.0
jedi                      0.19.1
Jinja2                    3.1.2
json5                     0.9.14
jsonpointer               2.4
jsonschema                4.19.1
jsonschema-specifications 2023.7.1
jupyter_client            8.4.0
jupyter_core              5.4.0
jupyter-events            0.8.0
jupyter-lsp               2.2.0
jupyter_server            2.8.0
jupyter_server_terminals  0.4.4
jupyterlab                4.0.7
jupyterlab-pygments       0.2.2
jupyterlab_server         2.25.0
MarkupSafe                2.1.3
matplotlib-inline         0.1.6
mistune                   3.0.2
nbclient                  0.8.0
nbconvert                 7.9.2
nbformat                  5.9.2
nest-asyncio              1.5.8
notebook_shim             0.2.3
numpy                     1.26.1
overrides                 7.4.0
packaging                 23.2
pandas                    2.1.1
pandocfilters             1.5.0
parso                     0.8.3
pexpect                   4.8.0
pickleshare               0.7.5
pip                       23.3.1
platformdirs              3.11.0
prometheus-client         0.17.1
prompt-toolkit            3.0.39
psutil                    5.9.6
ptyprocess                0.7.0
pure-eval                 0.2.2
pycparser                 2.21
Pygments                  2.16.1
python-dateutil           2.8.2
python-json-logger        2.0.7
pytz                      2023.3.post1
PyYAML                    6.0.1
pyzmq                     25.1.1
referencing               0.30.2
requests                  2.31.0
rfc3339-validator         0.1.4
rfc3986-validator         0.1.1
rpds-py                   0.10.6
scipy                     1.11.3
Send2Trash                1.8.2
setuptools                68.2.2
six                       1.16.0
sklearn                   0.0.post10
sniffio                   1.3.0
soupsieve                 2.5
stack-data                0.6.3
terminado                 0.17.1
tinycss2                  1.2.1
tornado                   6.3.3
traitlets                 5.11.2
types-python-dateutil     2.8.19.14
tzdata                    2023.3
uri-template              1.3.0
urllib3                   2.0.7
wcwidth                   0.2.8
webcolors                 1.13
webencodings              0.5.1
websocket-client          1.6.4
```
</details>

<details><summary>LaTeX details</summary>

+ LaTeX distribution (e.g. TeX Live 2020): MacTeX 2023
+ Installed LaTeX packages: [link](https://pastebin.com/qnmX0SZv)

</details>

<details><summary>FFMPEG</summary>

Output of `ffmpeg -version`:

```
ffmpeg version 6.0 Copyright (c) 2000-2023 the FFmpeg developers
built with Apple clang version 15.0.0 (clang-1500.0.40.1)
configuration: --prefix=/opt/homebrew/Cellar/ffmpeg/6.0_1 --enable-shared --enable-pthreads --enable-version3 --cc=clang --host-cflags= --host-ldflags='-Wl,-ld_classic' --enable-ffplay --enable-gnutls --enable-gpl --enable-libaom --enable-libaribb24 --enable-libbluray --enable-libdav1d --enable-libjxl --enable-libmp3lame --enable-libopus --enable-librav1e --enable-librist --enable-librubberband --enable-libsnappy --enable-libsrt --enable-libsvtav1 --enable-libtesseract --enable-libtheora --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libxvid --enable-lzma --enable-libfontconfig --enable-libfreetype --enable-frei0r --enable-libass --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libspeex --enable-libsoxr --enable-libzmq --enable-libzimg --disable-libjack --disable-indev=jack --enable-videotoolbox --enable-audiotoolbox --enable-neon
libavutil      58.  2.100 / 58.  2.100
libavcodec     60.  3.100 / 60.  3.100
libavformat    60.  3.100 / 60.  3.100
libavdevice    60.  1.100 / 60.  1.100
libavfilter     9.  3.100 /  9.  3.100
libswscale      7.  1.100 /  7.  1.100
libswresample   4. 10.100 /  4. 10.100
libpostproc    57.  1.100 / 57.  1.100
```
</details>

**Comment by Kyparos** [comment]
P.S. `pip3 -V` also looks fine

**Comment by Kyparos** [comment]
P.S.S on python 3.11 works just fine

**Comment by naveen521kk** [comment]
We still haven't released a version which will work on Python 3.12, yet. See https://github.com/ManimCommunity/manim/pull/3395

**Comment by behackl** [comment]
Should work now as intended with the fresh release. Closing this, but please report back in case the installation does not work!
